notebooks/week5/00.create_source_data.py

```
@@ -0,0 +1,126 @@
+
+
+# COMMAND ----------
+
+existing_ids = set(int(id) for id in combined_set['id'])
+
+# COMMAND ----------
+
+# Define function to create synthetic data without random state
+def create_synthetic_data(df, num_rows=100):
+    synthetic_data = pd.DataFrame()
+    
```

Ruff (0.8.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)